pkg/test/resourcefixture/contexts/register.go

```diff
@@ -200,6 +227,31 @@ func (rc ResourceContext) Delete(ctx context.Context, _ *testing.T, u *unstructu
 		if !deleted {
 			return fmt.Errorf("did not delete logging.cnrm.cloud.google.com/LoggingLogMetric '%v'", u.GetName())
 		}
+	case schema.GroupKind{Group: "gkehub.cnrm.cloud.google.com", Kind: "GKEHubFeatureMembership"}:
+		m, err := gkehub.NewGkeHubModel(ctx, &config.ControllerConfig{
+			HTTPClient: httpClient,
+		})
+		if err != nil {
+			return err
+		}
+		a, err := m.AdapterForObject(ctx, c, u)
+		if err != nil {
+			return err
+		}
+		found, err := a.Find(ctx)
+		if err != nil {
+			return err
+		}
+		if !found {
+			return fmt.Errorf("gkehub.cnrm.cloud.google.com/GKEHubFeatureMembership '%v' not found", u.GetName())
+		}
```

Comment:
This seems duplicate with Line 171 - 187. Can you refactor a little to avoid the duplication?